roles/ceph-common/defaults/main.yml

```diff
@@ -277,7 +277,6 @@ mds_max_mds: 3
 ## Rados Gateway options
 #
 #radosgw_dns_name: your.subdomain.tld # subdomains used by radosgw. See http://ceph.com/docs/master/radosgw/config/#enabling-subdomain-s3-calls
-radosgw_frontend: civetweb # supported options are 'apache' or 'civetweb', also edit roles/ceph-rgw/defaults/main.yml
```

Comment:
The ``radosgw_frontend`` var is being used in the the ceph.conf template. Removing this will cause the template to fail to render.

https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/templates/ceph.conf.j2#L116